spec/unit/action/install_chef/base_spec.rb

```diff
@@ -48,55 +48,23 @@
   end
 
   context "#perform_action" do
-    context "when chef is already installed on target at the correct minimum version" do
-      before do
-        expect(install.target_host).to receive(:installed_chef_version).and_return ChefApply::Action::InstallChef::Base::MIN_CHEF_VERSION
-      end
+    context "when chef is already installed on target" do
       it "notifies of success and takes no further action" do
+        expect(install).to receive(:check_minimum_chef_version!).with(install.target_host).and_return(:minimum_version_met)
         expect(install).not_to receive(:perform_local_install)
         install.perform_action
       end
     end
 
-    context "when chef is already installed on target at a version that's too low" do
-      before do
-        expect(install.target_host).to receive(:installed_chef_version).
-          and_return Gem::Version.new("12.1.1")
-      end
-      # 2018-05-10  pended until we determine how we want auto-upgrades to behave
-      xit "performs the upgrade" do
+    context "when chef is not already installed on target" do
+      it "should invoke perform_local_install" do
+        expect(install).to receive(:check_minimum_chef_version!).with(install.target_host).and_return(:client_not_installed)
```

Comment:
What are y'all's opinion on bumping long expectation stubs to multiple lines to make the difference pop?

```
expect(install).to receive(:check_minimum_chef_version!).with(install.target_host)
               .and_return(:client_not_installed)
```